src/main/asciidoc/reference/r2dbc-repositories.adoc

```diff
@@ -118,6 +118,9 @@ interface ReactivePersonRepository extends ReactiveSortingRepository<Person, Lon
 The annotated query uses native bind markers, which are Postgres bind markers in this example.
 ====
 
+Note that the columns of a select used in a `@Query` annotation must match the names generated by the `NamingStrategy` for the respective property.
```

Comment:
Change to:

Note that the columns of a select statement used in a `@Query` annotation must match the names generated by the `NamingStrategy` for the respective property.